src/index.js

```diff
@@ -450,6 +457,22 @@ const onMapLoad = async () => {
         }
       ],
       statusOptions,
+      searchOptions: {
+        fuzzySearch: {
+          // https://listjs.com/docs/fuzzysearch/
+          location: 0,
+          distance: 999999, // we don't care where the matched word is
+          threshold: 0.4,
+          multiSearch: true
+        },
```

Comment:
```suggestion
```

I think we want to just use the default exact match search rather than a fuzzySearch.
Reasoning: At a threshold of 0.4, "bilingual" matches 6 locations, should only match 3 locations, and misses 1 location. So this is definitely too high of a threshold. 

At a threshold of 0.3, "bilingual" matches 2 of 3 sites which include the word "bilingual" - this is the same outcome as the "exact" match, list.js search functions seem to be unable to match "bilingual" in the string "bilingual/bicultural." No version of fuzzy search catches "bilingual" in the string "bilingual/bicultural," so this is some character encoding issue maybe inherent to list.js. 

So why not go with fuzzySearch at a threshold of 0.3? Well...
If you search for zip code "55406" at threshold 0.3, you get over 100 results. It matches 55404, 55405, 55406, 55407, etc. - not useful for the users. There should be only 16 matches, which is accomplished at a threshold of 0.2, and in an "exact" match. 

So why not use fuzzySearch with a threshold of 0.2?
Ideally, we would want to effectively catch typos. This is a hard problem. If we set the threshold to 0.2 and search for "diper" instead of "diaper," we get 3 matching locations. We should get 56 locations. I think it is confusing to match a very small subset of the locations that have diapers, and would be less confusing to just NOT accommodate typos for now, until we can find a better solution for search.